lib/cli.rb

```diff
@@ -587,7 +591,83 @@ def self.check_port_validity(port)
     end
   end
 
+  desc "Inspect container image"
+  long_desc <<-LONGDESC
+    Inspect container image and generate system descripton from inspected data.
+
+    Multiple scopes can be passed as comma-separated list. If no specific scopes
+    are given, all scopes are inspected.
 
+    Available scopes: #{AVAILABLE_SCOPE_LIST}
+  LONGDESC
+  arg "IMAGENAME"
+  command "inspect-container" do |c|
+    supports_filtering(c)
+    define_inspect_command_options(c)
+    c.switch ["docker", :d], required: true, negatable: false,
+      desc: "Inspect a docker container"
+
+    c.action do |global_options,options,args|
```

Comment:
Unused block argument - `global_options`. If it's necessary, use `_` or `_global_options` as an argument name to indicate that it won't be used.<br>Space missing after comma.